examples/custom-client-cxx/main.cc

```diff
@@ -50,6 +51,13 @@ int main(int argc, char **argv) {
     cfg_dirs = AkliteClient::CONFIG_DIRS;
   }
 
+  if (env.end() != env.find("AKLITE_LOCAL_UPDATE_PATH")) {
```

Comment:
I would rather add new sub-command(s) or even a new executable for offline update. The online daemon mode and the offline non-daemon mode/update differ in subtle ways, hence it's better not to mix their implementation within a single context.

Reply:
IMO, it's worth adding a new executable that would provide customers with an example of a cli update client (non-daemon) that can perform both online and offline update.

Reply:
There are diffs (no need in the env. var., no reboot, etc). However there is no need in a new executable indeed. Please, see my comment in slack (as well as corresponding jira tickets) and sorry for misunderstanding :).